physicalplan/binary/table.go

```diff
@@ -0,0 +1,128 @@
+package binary
+
+import (
+	"fmt"
+
+	"github.com/prometheus/prometheus/promql/parser"
+	"github.com/thanos-community/promql-engine/physicalplan/model"
+)
+
+type sample struct {
```

Comment:
Is there a reason for not using `promql.Point`? 🙂

Reply:
Hm, no good reason, but let's maybe keep the package self-contained. It's a small struct anyway.